### DISCUSSION TITLE
New-component: Accordion -> Asking for opinion

### DISCUSSION THREAD

**Discussion by a-gonzalo** [sqlpage/SQLPage#547] [Show and tell]
Hello everyone,

I'm working on adapt the (last?) component of Tabler: Accordion:

![image](https://github.com/user-attachments/assets/a1b3caa8-b230-4c72-ba6c-7849100d75b8)

The actual implementation, renders the accordion correctly and It have the correct functionality of the collapse button:

![image](https://github.com/user-attachments/assets/b998004b-65a6-4197-8041-ba0c2d84982d)

And this is accordion.handlebars:

```handlebars
<div class="accordion" id="{{default accordion_id 'accordion_default'}}">
    {{#each_row}}
    <div class="accordion-item">
        <h2 class="accordion-header" id="heading-{{@row_index}}">
            <button class="accordion-button" type="button" data-bs-toggle="collapse"
                data-bs-target="#collapse-{{@row_index}}" aria-controls="collapse-{{@index}}">
                {{title}}
            </button>
        </h2>
        <div id="collapse-{{@row_index}}" class="accordion-collapse collapse {{#if (eq @row_index 0)}}show{{/if}}"
            aria-labelledby="heading-{{@row_index}}" data-bs-parent="#{{default accordion_id 'accordion_default'}}">
            <div class="accordion-body">
                {{content}}
            </div>
        </div>
    </div>
    {{/each_row}}
</div>
```

I'm opening to ask:

The currently implementation of Tabler is being loaded into a .card>.card-body. Should the accordion be embedded by default into a card or should it be an individual div and let people embed it by themselves?

Edit: Also, if there's any Tabler component that you want to replicate into SQLPage, let me know!

**Comment by a-gonzalo** [comment]
I noticed now that I'm not letting load Markdown content into the "content" property. I'm leaving this comment as a self-note to implement it asap

**Comment by lovasoa** [comment]
Wow, this looks great, thank you @Ajotah98 ! 

I think it looks better without the double border.

Like this

![image](https://github.com/user-attachments/assets/f6e7ee1d-cd9b-4299-8485-2aad6734f656)


Not like this

![image](https://github.com/user-attachments/assets/1b7963c0-33b1-4f17-8b6b-54bff7a3275b)

**Comment by lovasoa** [comment]
About other components, I think one that would be very useful for dashboards is the big_number, that highlights one key figure, and potentially its evolution.

Like [this from the tabler demo](https://tabler.io/preview)

![image](https://github.com/user-attachments/assets/fbdcc60f-b31a-40db-b153-c4a232422764)

Or like [that from evidence.dev](https://docs.evidence.dev/components/big-value/)

![image](https://github.com/user-attachments/assets/fe27696d-3afc-4a4e-9f88-930580973275)

**Comment by lovasoa** [comment]
Another one that I find useful in dahsboards is the "tables with a jauge", that associates a textual element to a numerical value, and represents the magnitude of the numerical value. It looks like multiple bar charts inline in the page.

![image](https://github.com/user-attachments/assets/7af539c1-54fa-4797-b401-95b4443847f7)

They call it "dimension grid" in evidence: https://docs.evidence.dev/components/dimension-grid/

**Reply by a-gonzalo** [reply]
I really like this design. I didn't know about evidence.dev, I'll peek to see how can I implement it!
